scripts/utils.js

```diff
@@ -176,6 +176,48 @@ async function updateDeploymentScriptVersion(nextVersion) {
   );
 }
 
+/**
+ * @param {string} importName
+ * @returns {[string, string]} [packageName, importPath]
+ */
+const getPackageNameFromImportName = (importName) => {
```

Comment:
nit: ASTExplorer calls the package name in `"` the "import specifier". I think that is more apt here than `importName`:

```suggestion
const getPackageNameFromImportSpecifier = (importSpecifier) => {
```